### ISSUE TITLE
cached widget models shouldn't contains "additionalData"

### ISSUE THREAD

**Issue by JohnsonJohnsonYuan** [nopSolutions/nopCommerce#3071]
nopCommerce version: 4.01

Steps to reproduce the problem: 
~\Presentation\Nop.Web\Factories\WidgetModelFactory.cs

The additionalData is always cached,  I think we shouldn't include additionalData property when initialize componentArguments variable,
```c#
           //add widget zone to view component arguments
            var componentArguments = new RouteValueDictionary
            {
                { "widgetZone", widgetZone },
                { "additionalData", additionalData }         // ?????????? always cached
            };

            var cachedModel = _cacheManager.Get(cacheKey, () =>
            {
                //model
                var model = new List<RenderWidgetModel>();

                var widgets = _widgetService.LoadActiveWidgetsByWidgetZone(widgetZone, _workContext.CurrentCustomer, _storeContext.CurrentStore.Id);
                foreach (var widget in widgets)
                {
                    model.Add(new RenderWidgetModel
                    {
                        WidgetViewComponentName = widget.GetWidgetViewComponentName(widgetZone),
                        WidgetViewComponentArguments = componentArguments
                    });
                }
                return model;
            });

```

**Comment by AndreiMaz** [comment]
it's be design. If you experience some issue with it, please provide a list of steps to reproduce the issue

**Comment by RuS984** [comment]
Fixed by #3138.

Closed #3071.
